actions/approve-pr/src/enable-auto-merge.ts

```diff
@@ -4,12 +4,18 @@ import { Octokit, Graphql } from '@dfinity/action-utils';
 export interface EnableAutoMergeOptions {
   owner: string;
   repo: string;
-  mergeMethod: string;
+  mergeMethod: MergeMethod;
```

Comment:
This parameter was already used further down, but it wasn't being supplied a value.